gnucash/gnome/dialog-stock-editor.h

```diff
@@ -0,0 +1,31 @@
+/********************************************************************\
+ * window-autoclear.h -- the autoclear window                       *
+ * Copyright (C) 2010 Cristian KLEIN                                *
+ *                                                                  *
+ * This program is free software; you can redistribute it and/or    *
+ * modify it under the terms of the GNU General Public License as   *
+ * published by the Free Software Foundation; either version 2 of   *
+ * the License, or (at your option) any later version.              *
+ *                                                                  *
+ * This program is distributed in the hope that it will be useful,  *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the    *
+ * GNU General Public License for more details.                     *
+ *                                                                  *
+ * You should have received a copy of the GNU General Public License*
+ * along with this program; if not, contact:                        *
+ *                                                                  *
+ * Free Software Foundation           Voice:  +1-617-542-5942       *
+ * 51 Franklin Street, Fifth Floor    Fax:    +1-617-542-2652       *
+ * Boston, MA  02110-1301,  USA       gnu@gnu.org                   *
+\********************************************************************/
+
+#ifndef WINDOW_STOCKEDITOR_H
+#define WINDOW_STOCKEDITOR_H
+
+typedef struct _StockEditorWindow StockEditorWindow;
+
+StockEditorWindow *stockEditorWindow (GtkWidget *parent, Account *account);
+void gnc_ui_stockeditor_window_raise (StockEditorWindow * autoClearData);
```

Comment:
Not realy sure why you have done this as it seems to be not required unless you intend using the StockEditorWindow.
Have a look at the dialog-imap-editor
void gnc_stock_editor_dialog (GtkWidget *parent, Account *account);

Reply:
Amended. A future possible extension will be to launch the dialog prepopulated with current stock txn details. Not sure how safe it'll be to recreate/amend existing txn to have new data. The *safest* approach would be to destroy the previous txn and recreate.